practices/handover.md

```diff
@@ -0,0 +1,72 @@
+---
+slug: handover
+stages:
+  - development
+short_description: 
+tags:
+  - efficiency
+  - sustainability
+  - knowledge sharing
+  - knowledge transfer
+  - documentation
+keywords:
+  - agile-handover
+  - knowledge transfer
+  - team building
+  - communication
+  - support 
+covered_by_dx_scanner: false
+---
+
+# Handover
+
+**TL;DR**
+
+Transferring projects from one team to another is often seen as a mammoth task.  It requires a complete transfer of roles and responsibilities. However, following set protocols, having strong communication and reciprocated motivation makes this whole process a lot smoother.
+
+## What Is a Handover?
+
+A handover is a process of transferring the roles and responsibilities of a project from one team/individual to another. Typically, this transfer takes place over the duration of a couple of months, in order to ensure that the receiving team/individual has fully grasped the working of the project, and their roles corresponding to the same. 
+
+## What Is Handover Documentation?
+
+Handover documentation refers to the resources, notes and documents compiled or passed-on by staff members/teams who are about to leave thier current position (either temporarily or permanently). They are 'handed over' to the team/individual that will be succeeding them, in order to help them carry out their newly acquired duties with ease.
+
+## Handover Procedure
```

Comment:
```suggestion
### Handover Procedure
```
The following lines seem to describe what handover is. Thus, I would suggest using a smaller headline.